code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -206,7 +205,6 @@
 			dropItemToGround(s_store, TRUE) //It makes no sense for your suit storage to stay on you if you drop your suit.
 		if(wear_suit.breakouttime) //when unequipping a straightjacket
 			REMOVE_TRAIT(src, TRAIT_RESTRAINED, SUIT_TRAIT)
-			REMOVE_TRAIT(src, TRAIT_HANDS_BLOCKED, SUIT_TRAIT)
```

Comment:
I assume the removals like this are handled elsewhere?

Reply:
`TRAIT_RESTRAINED` is now a source of `TRAIT_HANDS_BLOCKED`